lib/optional_query_cache/version.rb

```diff
@@ -0,0 +1,11 @@
+require 'active_record'
```

Comment:
Rather than requiring active record, you can just declare the types:

    module ActiveRecord
    	class QueryCache
    	end
    end